### DISCUSSION TITLE
Closing Cylindrical coordinates

### DISCUSSION THREAD

**Discussion by electroflow** [thliebig/openEMS-Project#196] [Q&A Usage] [answered]
Is there a way to close the cylindrical coordinates such that there is no hole in the middle of the simulation domain and no "cakeslice" missing from the simulation domain?

**Comment by thliebig** [comment, marked as answer]
I would need to check, but if I remember correctly, you only need to include r=0 to include the center.
And if you want to have a fully closed cylindrical mesh, then the alpha mesh should cover the entire 0..2*pi (or e.g. -pi..+pi, etc).

**Reply by electroflow** [reply]
Thank you, I had been confused by the depiction in paraview in which a section is missing but I think the domains are indeed connected.
